tests/jballerina-unit-test/src/test/resources/test-src/types/table/record-constraint-table-value.bal

```diff
@@ -492,6 +493,19 @@ public function testSpreadFieldInConstructor() {
     tb3.toString());
 }
 
+function testTableConstructExprVar() {
+    string s1 = "id";
+    string s2 = "employed";
+
+    var v1 = table [
+        {name: "Jo"},
+        {[s1]: 2},
+        {[s2]: false}
+    ];
+
```

Comment:
Tried the following code, seems it gives a runtime error

```ballerina
map<int> m = {name: 1, b: 2};
    var v2 = table [
        {name: "Jo"},
        {...m}
    ];
```

Should the table's constraint type be `record{|string|int name; ...int|} `here?

Reply:
We have a separate issue for that one. Can fix separately
https://github.com/ballerina-platform/ballerina-lang/issues/33297